docs/sdldirectives.rst

```diff
@@ -0,0 +1,306 @@
+Schema Directives
+=================
+
+Adding Behaviour
+----------------
+
+Schema Definition Language (SDL) allows you to define your graphql types in a declarative manner without using code.  However you still need to wire in all the
+logic that backs those types and their fields.
+
+Schema directives allow you to do this.  You can place directives on SDL elements and then write the backing logic
```

Comment:
maybe make it clear what the difference is between this apporach and just using normal RuntimeWiring: something along the line of using directive as some kind of indirection or additional layer?

Reply:
will do